applications/FluidDynamicsApplication/custom_utilities/post_process_utilities.h

```diff
@@ -0,0 +1,46 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Miguel Angel Celigueta
+
+
+#ifndef POST_PROCESS_UTILITIES_H
+#define POST_PROCESS_UTILITIES_H
+
+#include "includes/define.h"
+#include "includes/variables.h"
+
+namespace Kratos {
+
+    class PostProcessUtilities {
```

Comment:
I'm strongly against adding even more files to the application. We can add either add the `ComputeFlow` method to the process as a private method or to the utilities @sunethwarna created some days ago (unfortunately these are not merged yet).